api_v3/services/ExportCsvService.php

```diff
@@ -94,13 +94,26 @@ public static function generateCsvPath($id, $ks)
 		// KS verification - we accept either admin session or download privilege of the file
 		if(!$ks->verifyPrivileges(ks::PRIVILEGE_DOWNLOAD, $id))
 			KExternalErrors::dieError(KExternalErrors::ACCESS_CONTROL_RESTRICTED);
-		
-		$partner_id = kCurrentContext::getCurrentPartnerId();
-		$folderPath = "/content/exportcsv/$partner_id";
-		$fullPath = myContentStorage::getFSContentRootPath() . $folderPath;
-		$file_path = "$fullPath/$id";
-		
-		return $file_path;
+
+		$partnerId = kCurrentContext::getCurrentPartnerId();
+		$folderPath = "/content/exportcsv/$partnerId";
```

Comment:
Try to make this a bit more simple
The only diff here should be how you create the folderPath it will always be 
/content/exportcsv/
and then the diff is if the inner folder name is partnerId or the Scattered